### PR TITLE
Adding styling for images in panel overlay, improving overlay markup/css

### PR DESCRIPTION
**Pull request by rhamilto**
font icon
screen shot 2017-04-11 at 9 38 47 am

img
screen shot 2017-04-11 at 9 39 07 am

### CODE REVIEW

**Comment by dtaylor113** [comment]
Would be nice to see a screenshot in the PR, just to be able to quickly see the improvements.

File: dist/origin-web-catalogs.js [hunk 1]
```diff
@@ -7,13 +7,13 @@ webpackJsonp([ 0, 1 ], [ function(e, t) {
...
...a fa-check success-check"></span>\n    </div>\n    <div class="sub-title center">\n      Continue to your project to view your application.\n    </div>\n    <div class="launch-service">\n      \x3c!-- TODO: Correctly set baseProjectUrl --\x3e\n      <a ng-href="{{$ctrl.selectedProject | projectUrl : $ctrl.baseProjectUrl}}" class="btn btn-primary order-btn">\n        View Project\n      </a>\n    </div>\n  </div>\n  <div ng-if="$ctrl.error">\n    <div class="title">Error <span class="fa fa-times text-danger"></span></div>\n    <div class="sub-title center">\n      <span ng-if="$ctrl.error.data.message">\n        {{$ctrl.error.data.message | upperFirst}}\n      </span>\n      <span ng-if="!$ctrl.error.data.message">\n        An error occurred creating the application.\n      </span>\n    </div>\n    \x3c!-- TODO: Improve error message presentation --\x3e\n    <ul ng-if="$ctrl.error.failure.length" class="failure-messages">\n      <li ng-repeat="failure in $ctrl.error.failure">\n        {{failure.data.message}}\n      </li>\n    </ul>\n  </div>\n  <div>\n    <a class="close-href" href="" ng-click="$ctrl.closePanel()">Close</a> this panel to browse other services.\n  </div>\n</div>\n';
```
Comment (rhamilto) [inline dist/origin-web-catalogs.js]: @jeff-phillips-18, I've got an issue when running `npm start build`.  &lt; and &gt; characters as part of an html comment are being escaped as hex codes, which results in invalid dist.  Have you seen this issue before?  I'm running the same node (v4.8.2) and npm (v2.15.11) versions as @spadgett, yet he can build from my source without the issue.

Comment (jeff-phillips-18) [inline dist/origin-web-catalogs.js]: @rhamilto what version of webpack do you have?

File: src/styles/order-service.less [hunk 2]
```diff
@@ -12,61 +12,65 @@
...
     .learn-more-link {
       font-size: @font-size-small;
       white-space: nowrap;
     }
   }
   .order-service-tags {
-    margin-left: -5px;
+    margin-top: 5px;
     .tag {
       margin-left: 5px;
```
Comment (spadgett) [inline src/styles/order-service.less]: If the tags wrap, will this leave the first tag on the second line indented in?

Comment (rhamilto) [inline src/styles/order-service.less]: Indeed it will.  Lemme fix that.

**Pull request merged**